src/components/signpost.js

```diff
@@ -7,20 +7,58 @@ import styles from './signpost.module.scss';
 
 export const directions = [
   {
-    emoji: "👈",
-    label: "Library",
+    emoji: "🏢",
```

Comment:
I changed this to a different icon a while ago at the request of a few folks and then got asked to put it back to the 👈🏻. Lets see what happens 😄

Reply:
Hahaha, yeah, I was gonna point this out too, as I did last time. 😄 It may be time to just change this format though.